docs/reference/compatibility.txt

```diff
@@ -531,18 +540,26 @@ provided compatibility with wherever possible.
    * - ``ActiveModel``
      - |checkmark|
 
-   * - ``ActiveRecord``
-     - |x|
-
    * - ``ActiveStorage``
      - |x| [#rails-activerecord-dependency]_
 
-.. [#rails-activerecord-dependency] Depends on ``ActiveRecord``
+.. [#rails-actioncable-dependency] There is currently no MongoDB adapter for
+   ``ActionCable``, however any existing adapter can be used successfully in conjunction
```

Comment:
I'd suggest to add "any existing adapter _such as Redis_" to clarify the meaning here